index.html

```diff
@@ -5,9 +5,37 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
     <title>mobileFirst</title>
 </head>
 <body>
+    <nav>
+        <header>My Logo</header>
+        <img src="Images/menu-button.png" alt="">  
+    </nav>
```

Comment:
- [ ] Please make sure to fully understand the last reviewer comment about the nav, the logo should be wrapped in an `a` tag, if you take a look at websites such as youtube or Facebook, and you pressed the logo it will take you to the home page of the website since it is the virtual identity, secondly, it was not a good choice to wrap the logo in a header tag since The <header> element represents a container for introductory content or a set of navigational links, the nav tag supposes to wrap the hamburger button only not all the header section since it represents a set of navigational links that suppose to be inside the menu when you press the hamburger button.